src/compiler/factory/nodeFactory.ts

```diff
@@ -510,6 +511,7 @@ export function addNodeFactoryPatcher(fn: (factory: NodeFactory) => void) {
  */
 export function createNodeFactory(flags: NodeFactoryFlags, baseFactory: BaseNodeFactory): NodeFactory {
     const update = flags & NodeFactoryFlags.NoOriginalNode ? updateWithoutOriginal : updateWithOriginal;

```

Comment:
This isn't needed for this PR, but if we instead use the new `setOriginal`, we could just rename `updateWithOriginal` to `update` and remove this conditional. But, maybe that's slower for some reason.

Reply:
Perf is good with this change so did it